native_client/README.rst

```diff
@@ -5,7 +5,6 @@ Building DeepSpeech Binaries
 If you'd like to build the DeepSpeech binaries yourself, you'll need the following pre-requisites downloaded and installed:
 
 
-* `Mozilla's TensorFlow r2.2 branch <https://github.com/mozilla/tensorflow/tree/r2.2>`_
 * `Bazel 2.0.0 <https://github.com/bazelbuild/bazel/releases/tag/2.0.0>`_
 * `General TensorFlow requirements <https://www.tensorflow.org/install/install_sources>`_
```

Comment:
Should this now mention 2.2 explicitly since we remove the version specifier on the line above?

Reply:
Well we still need Bazel version 2.0.0, so I think we should keep it or people will again start doing wrong stuff

Reply:
I meant that we should mention `TensorFlow 2.2 requirements` rather than just `TensorFlow requirements`.